### PR TITLE
📻 Allow Editor to support all the audio formats that we can play

### PR DESCRIPTION
**Pull request by norkans7**
Addresses https://github.com/rapidpro/rapidpro/issues/779

### CODE REVIEW

File: static/coffee/flows/controllers.coffee [hunk 1]
```diff
@@ -208,8 +208,8 @@ app.controller 'FlowController', [ '$scope', '$rootScope', '$timeout', '$log', '
         showDialog('Invalid Attachment', 'Attachments must be either video, audio, or an image.')
         return
 
-      if media_type == 'audio' and media_encoding != 'mp3'
-        showDialog('Invalid Format', 'Audio attachments must be encoded as mp3 files.')
+      if media_type == 'audio' and media_encoding not in ['mp3', 'm4a', 'wav', 'ogg', 'oga']
```
Comment (rowanseymour) [inline static/coffee/flows/controllers.coffee]: @norkans7 can you take a quick look at all the channel types that support audio attachments and see what formats they support? I think we should be aiming for a set of formats here that work on almost all channel types.

Comment (norkans7) [reply static/coffee/flows/controllers.coffee]: I did so before

Comment (rowanseymour) [reply static/coffee/flows/controllers.coffee]: Ok sorry to be a pain but can you make a table of who supports what? Including Whatsapp and Twitter. The ideal solution here is that we force users to upload attachments in formats that are going to work on the maximum number of channel types.

Comment (norkans7) [reply static/coffee/flows/controllers.coffee]: Okay

Comment (norkans7) [reply static/coffee/flows/controllers.coffee]: | Type | Allowed | Forbidden |
|------|---------|------------|
| Facebook | ALL(audio, video, image ,file) | None |
|Viber | text, picture, video, file, location, contact, sticker, carousel content and url | https://developers.viber.com/docs/api/rest-bot-api/#forbidden-file-formats |
| Telegram | ALL | None |
| Twilio | https://www.twilio.com/docs/sms/accepted-mime-types | None |

Comment (norkans7) [reply static/coffee/flows/controllers.coffee]: | type | MP3 | M4A | WAV | OGG | OGA |
|------|------|------|-----|------|------|
| Facebook | Yes | Yes |Yes|Yes|Yes|Yes|
| Viber | Yes | Yes |Yes|Yes|Yes|Yes|
| Telegram | Yes | Yes |Yes|Yes|Yes|Yes|
| Twilio | `No` | Yes |Yes|Yes|Yes|Yes|

**Pull request merged**